Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Procedural/ProceduralPrefabAsset.cpp

```diff
@@ -58,9 +59,10 @@ namespace AZ::Prefab
 
```

Comment:
should bumping the

serializeContext->Class<ProceduralPrefabAsset, AZ::Data::AssetData>()->Version(1)

make all the FBX re-build if their product asset (Procedural Prefab version) is updated?

maybe this is better than bumping all FBX files to rebuild?

Reply:
It would have to be included in the fingerprint generated by the job and the builder, which I doubt it currently is.  AP doesn't do any auto fingerprinting.  Also with proc prefabs, since they don't have their own job, we'd have to do a bit of work to have additional fingerprint info that's specific to the proc prefab, otherwise we'd still end up rebuilding all FBXs.